pages/common/tlmgr-option.md

```diff
@@ -0,0 +1,28 @@
+# tlmgr option
+
+> TeX Live settings manager.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- List all TeX Live settings:
+
+`tlmgr option showall`
+
+- List all currently set Tex Live settings:
+
+`tlmgr option show`
+
+- List all TeX Live settings as JSON array:
```

Comment:
I prefer `display` or `print` if it's not a list.
```suggestion
- Display all TeX Live settings as JSON array:
```

Reply:
Makes sense. I prefer `print`tough